src/ClientApp/aspnetcore-https.js

```diff
@@ -0,0 +1,33 @@
+// This script sets up HTTPS for the application using the ASP.NET Core HTTPS certificate
```

Comment:
Q: Braucht es dieses File überhaupt, wenn wir HTTPS in der Dev Umgebung abschalten?

Reply:
Ich würds drin lassen, falls jemand lokal etwas mit HTTPS testen muss.